ajax/libs/dagre/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "dagre",
+  "version": "0.7.4",
+  "filename": "dagre.min.js",
+  "description": "Graph layout for JavaScript",
+  "homepage": "https://github.com/cpettitt/dagre/wiki/",
+  "keywords": [
+    "graph",
+    "layout"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/cpettitt/dagre"
+  },
+  "license": "MIT",
+  "author": "Chris Pettitt <@cpettitt>",
```

Comment:
Maybe we can follow the format as the same as upstream. ref: https://github.com/cpettitt/dagre/blob/master/package.json#L5
`"author": "Chris Pettitt <cpettitt@gmail.com>"`